modules/apps/forms-and-workflow/dynamic-data-mapping/dynamic-data-mapping-service/src/main/resources/com/liferay/dynamic/data/mapping/dependencies/ddm/text-html.ftl

```diff
@@ -7,12 +7,22 @@
 	inputEditorName = "${namespacedFieldName}Editor"
 
 	fieldValue = paramUtil.getString(request, "${inputEditorName}", fieldValue)
+
+	toolbarSet = ""
```

Comment:
@Robert-Frampton that should work fine too - I tested that when I was making the fix and it works.